### DISCUSSION TITLE
Enable editing of file provisioned alerts

### DISCUSSION THREAD

**Discussion by Maelstromeous** [grafana/grafana#67576] [Alerting]
I would like the ability to mark file provisioned alerts as editable (just like dashboards). Currently, if you import said alerts and you want to tweak them, you have to essentially have two grafana instances where you have one that is unprovisioned and editable, export the JSON to then provision another instance with the file.

Annoyingly, there are two further issues at play:

1. While you have the option to copy the alert to enable editing to another folder, you are **unable to delete the original alert**, even via API, because you provisioned it via a file. This causes an issue where you have two alerts, one is slightly different. While you have the option to replace the provisioned alert with your new one via file provisioning by exporting all alerts, then editing manually with JSON, this is a PITA.
2. If you ever at all provision your local dev instance, you're screwed. You have no means to remove the alerts and essentially have to re-do all of your work to manually recreate all your alerts.

A workaround to this would be to add an "import" button for alerts, **in an "unprovisioned" state**. This would then mean you can export all your alerts to another instance, tweak them at will, then export them to be provisioned via file provisioning as normal.

**Comment by holmesb** [comment]
Yes if you're a gitops Grafana user, the new alerts aren't really ready yet IMO. Doesn't seem to be much curation of this board to tidy up duplicates. [#53490](https://github.com/grafana/grafana/discussions/53490). Also related to #66541
